client/components/happychat/composer.jsx

```diff
@@ -44,7 +44,11 @@ export const Composer = React.createClass( {
 				onMouseLeave={ this.scrollbleedUnlock }
 				>
 				<div className="happychat__message">
+					<label htmlFor="hc_message_textarea" style={{ display: "none"}}>
```

Comment:
My understanding is that it needs some text to read what the textarea is for.
It looks like an alternate could be to use `aria-label` within the textarea
See: https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html